README.md

```diff
@@ -27,13 +27,28 @@ After deploying this, the following steps must be completed.
 └── versions.tf
 ```
 
+
+
```

Comment:
Maybe a brief in-line comment that makes clearer that this section will be built automatically by a GitHub Action and should be touched otherwise?

Reply:
Added clarification here.